### DISCUSSION TITLE
Create PR instead of issue from a task list

### DISCUSSION THREAD

**Discussion by OndrejSpanel** [community/community#5171] [Projects and Issues]
When starting a work on a new feature, I most often create a PR, not an issue. Currently my workflow with task-lists is:

1. create a branch
2. push some trivial commit starting a work
3. create a PR
4. link PR from a task list

None of those tasks is automated yet. Creating issue is automatic, but issue cannot be converted to PR, therefore this does not help much.

Would it be possible to automate points 3. and 4. somehow, i.e. create a PR instead of an issue and let me choose a branch for the PR?

**Comment by douglaslassance** [comment]
I agree. Because both PR and issues can be shown in projects and because there is always a PR for any given issue, why even bother with issues at all. What @OndrejSpanel suggests would help to enable this issue-less workflow, but there is a couple of other additional points to consider. First, the newly created PRs should be drafts. Second, there should be available workflows to react to PRs going in and out of drafts. Last, it should be possible to assign yourself as a reviewer even though you created the PR.
